src/features/TodoList/TodoListItem.jsx

```diff
@@ -0,0 +1,55 @@
+import { useState } from 'react';
+import TextInputWithLabel from '../../assets/shared/TextInputWithLabel';
+
+function TodoListItem({ item, onCompleteTodo, onUpdateTodo }) {
+  const [isCompleted, setIsCompleted] = useState(false);
```

Comment:
This is unused so this needs to be cleaned up. There should also be no state for `isCompleted`.